docs/reference/openapi-rules.md

```diff
@@ -246,11 +246,22 @@ info:
   title: 'some title with <script>alert("You are Hacked");</script>',
 ```
 
-### openapi-tags-alphabetical
+### openapi-tags
 
-OpenAPI object should have alphabetical `tags`. This will be sorted by the `name` property.
+OpenAPI object should have non-empty `tags` array, and the tags names should be ordered alphabetically.
```

Comment:
I'm absolutely fine with making them recommended by default, but what bothers me is that enforcing alphabetical order might not always be intended, at least looking from Studio perspective, since you might want to have X tag be displayed first in the UI.
Either way, if you all are fine with that, I won't oppose.
I just want to make sure we all aware of possible side effects of this change.
@philsturgeon, @marbemac, @lottamus, what do you think folks?

Reply:
We should not validate tags are alphabetically ordered by default IMHO. In the past, we've used the ordering of tags to control how things are ordered in docs.

Reply:
Interesting! Ok, I opened this PR as a conversation starter and was wondering if we had reasons for doing things this way. Intentionally out of order tags is not something I’d considered a feature but thanks for sharing!

Reply:
@philsturgeon I think we could still make `openapi-tags` rule recommended, right?
Sounds like a good default.

Reply:
Yeah I think there is still value in the `openapi-tags` rule being a recommended rule. The issue is that this PR changes it to do more than just check for presence of tags - it's also enforcing alphabetical order.

What about, as part of Spectral v5, we make `openapi-tags` recommended and add a new `openapi-tags-alphabetical` rule (that is not in the recommended set) that does what was added here: https://github.com/stoplightio/spectral/pull/657/files#diff-ce8b5719463f4eb8546e80d8f0a8e6d0R222-R228?

Reply:
@marbemac `openapi-tags-alphabetical` already exists and is not recommended already, so we can just switch `openapi-tags` to be recommended in develop and job done! Will do now.